utils/build-llvm-local.sh

```diff
@@ -27,20 +27,26 @@ PYTHON_ROOT=`pip3 show pybind11 | grep Location | awk '{print $2}'`
 
 mkdir -p $LLVM_DIR/$BUILD_DIR
 mkdir -p $LLVM_DIR/$INSTALL_DIR
+LLVM_ENABLE_RTTI=${LLVM_ENABLE_RTTI:OFF}
```

Comment:
I definitely wouldn't do this - did someone (iree folks?) ask you to turn off RTTI?

Reply:
> Why disable by default? Did someone (iree folks?) ask you to turn off RTTI?

It is indeed related to interfacing with IREE, where disabling RTTI would help with delivering the end-to-end flow simpler and quicker.